apps/mobile/src/components/avatar-icon.tsx

```diff
@@ -0,0 +1,88 @@
+import AlienIcon from '@/assets/avatar-icons/alien.svg';
```

Comment:
I think that these should go in `packages/ui`? 

We can make a new task to move them

Reply:
I created an issue to make the AvatarIcon in the UI library for mobile. @pete-watters is correct, this is already used in the extension and should be part of the UI lib.

https://github.com/leather-io/issues/issues/290

Reply:
We can just pull these out with that issue so it doesn't delay getting this merged.

Reply:
Thanks! Yes lets not block merging this, I just wanted to mention it